### PR TITLE
Docker postgre yes

### PR DESCRIPTION
**Pull request by Axeleron007**
Please review and merge with master

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/elsa-workflows/elsa-core?pullRequest=1207) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/elsa-workflows/elsa-core?pullRequest=1207) before we can accept your contribution.<br/><br/>:x: Axeleron007 [sign now](https://cla.dotnetfoundation.org/elsa-workflows/elsa-core?pullRequest=1207)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/elsa-workflows/elsa-core?pullRequest=1207) it.</sub>

File: src/activities/webhooks/Elsa.Activities.Webhooks/Elsa.Activities.Webhooks.csproj [hunk 1]
```diff
@@ -12,6 +12,10 @@
         <PackageTags>elsa, workflows, webhooks</PackageTags>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Compile Remove="Extensions\WebhookServiceCollectionExtensions.cs" />
+    </ItemGroup>
+
```
Comment (sfmskywalker) [inline src/activities/webhooks/Elsa.Activities.Webhooks/Elsa.Activities.Webhooks.csproj:18]: I guess we can remove this block?

File: src/activities/webhooks/Elsa.Activities.Webhooks/Endpoints/WebhookDefinitions/Post.cs [hunk 2]
```diff
@@ -46,7 +46,7 @@ public async Task<ActionResult<WebhookDefinition>> Handle([FromBody] SaveWebhook
                 webhookDefinition = new WebhookDefinition
                 {
                     Id = !string.IsNullOrWhiteSpace(webhookId) ? webhookId : _idGenerator.Generate(),
-                };
+                };
```
Comment (sfmskywalker) [inline src/activities/webhooks/Elsa.Activities.Webhooks/Endpoints/WebhookDefinitions/Post.cs:49]: This seems like a non-change (added an accidental space it seems).

File: src/activities/webhooks/Elsa.Webhooks.Persistence.MongoDb/Services/DatabaseInitializer.cs [hunk 3]
```diff
@@ -25,13 +25,23 @@ public async Task ExecuteAsync(CancellationToken cancellationToken = default)
 
         private async Task CreateWebhookDefinitionsIndexes(CancellationToken cancellationToken)
         {
+            //var builder = Builders<WebhookDefinition>.IndexKeys;
+            //var tenantKeysDefinition = builder.Ascending(x => x.TenantId);
+            //var nameKeysDefinition = builder.Ascending(x => x.Name);
+            //var webhookIdDefinition = builder.Ascending(x => x.Id);
+            //var collection = _mongoContext.WebhookDefinitions;
+            //await CreateIndexesAsync(_mongoContext.WebhookDefinitions, cancellationToken, tenantKeysDefinition, webhookIdDefinition, nameKeysDefinition);
+            //await collection.Indexes.CreateOneAsync(new CreateIndexModel<WebhookDefinition>(webhookIdDefinition, new CreateIndexOptions { Unique = true }), cancellationToken: cancellationToken);
```
Comment (sfmskywalker) [inline src/activities/webhooks/Elsa.Webhooks.Persistence.MongoDb/Services/DatabaseInitializer.cs:34]: Do we need to keep these commented lines?

File: src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Documents/WebhookDefinitionDocument.cs [hunk 4]
```diff
@@ -4,7 +4,7 @@ namespace Elsa.Webhooks.Persistence.YesSql.Documents
 {
     public class WebhookDefinitionDocument : YesSqlDocument
     {
-        public string WebhookDefinitionId { get; set; } = default!;
+        public string WebhookId { get; set; } = default!;
```
Comment (sfmskywalker) [inline src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Documents/WebhookDefinitionDocument.cs:7]: Let's rename this to `DefinitionId` to be consistent with the name used in `WorkflowDefinitionDocument`?

File: src/designer/elsa-workflows-studio/src/components/screens/webhook-definition-editor/elsa-webhook-definition-editor-screen/elsa-webhook-definition-editor-screen.tsx [hunk 5]
```diff
@@ -94,7 +94,7 @@ export class ElsaWebhookDefinitionEditorScreen {
     this.saving = true;
 
     try {
-      
+      debugger
```
Comment (sfmskywalker) [inline src/designer/elsa-workflows-studio/src/components/screens/webhook-definition-editor/elsa-webhook-definition-editor-screen/elsa-webhook-definition-editor-screen.tsx:97]: Let's remove this.

File: src/persistence/Elsa.Persistence.YesSql/Stores/YesSqlStore.cs [hunk 6]
```diff
@@ -41,14 +41,14 @@ public virtual async Task SaveAsync(T entity, CancellationToken cancellationToke
         {
             await _semaphore.WaitAsync(cancellationToken);
 
-            try
+            try
```
Comment (sfmskywalker) [inline src/persistence/Elsa.Persistence.YesSql/Stores/YesSqlStore.cs:44]: This file contains changes only in accidentally added whitespace. Let's clean that.

**Review by sfmskywalker — APPROVED** [review]
Very nice! Just a few small aesthetic comments.

File: src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Documents/WebhookDefinitionDocument.cs (hunk 4 above)
Comment (Axeleron007) [reply src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Documents/WebhookDefinitionDocument.cs]: WebhookId renamed to DefinitionId

**Comment by Axeleron007** [comment]
Hi Sipke, I made code clean up based on your comments. The latest version is in the branch. Is it possible to merge this with Master please?

File: src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Migrations.cs [hunk 7]
```diff
@@ -11,7 +11,7 @@ public int Create()
         {
             SchemaBuilder.CreateMapIndexTable<WebhookDefinitionIndex>(
                 table => table
-                    .Column<string>(nameof(WebhookDefinitionIndex.WebhookDefinitionId))
+                    .Column<string?>(nameof(WebhookDefinitionIndex.DefinitionId))
```
Comment (sfmskywalker) [inline src/activities/webhooks/Elsa.Webhooks.Persistence.YesSql/Migrations.cs:14]: Isn't `DefinitionId` a required value?

**Review by sfmskywalker — APPROVED** [review]

**Pull request merged**